packages/package_info_plus/package_info_plus/pubspec.yaml

```diff
@@ -29,18 +29,18 @@ dependencies:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  http: ^0.13.0
```

Comment:
Is there a reason why you're tightening these three constraints to the latest version? This may cause problems for projects that contain other packages that have pinned a version for some reason. Unless there's a reason to migrate forward (e.g. you rely on a feature in meta v1.8), it's better to keep these looser.

Reply:
The next release for Plus plugins will be a major bump because of some breaking changes (dropping support of Android Gradle Plugin < 4.2). Due to this I would like to bump dependencies versions as well to stimulate people move forward, considering that the release will be breaking anyway.